src/index.ts

```diff
@@ -25,6 +25,7 @@ export * from "./reserve"
 export * from "./user"
 export * from "./idl"
 export * from "./types"
+export * as gov from "./gov"
```

Comment:
```suggestion
export * as governance from "./gov"
```
I would just change the export to `governance` for verbosity/clarify for the users' benefit.